docs/04-operation-guides/operations/obsv-01-access-logs.md

```diff
@@ -1,23 +0,0 @@
----
-title: Access Kyma application logs
```

Comment:
Is that document covered somewhere?

Reply:
Issue created to add the content to the HP -> https://github.tools.sap/kyma/backlog/issues/4814